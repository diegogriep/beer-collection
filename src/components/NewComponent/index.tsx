import * as S from './styles'

const NewComponent = () => (
  <S.Wrapper>
    <h1>NewComponent</h1>
  </S.Wrapper>
)

export default NewComponent
