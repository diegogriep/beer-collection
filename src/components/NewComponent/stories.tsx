import { Meta, StoryObj } from '@storybook/react'
import NewComponent from '.'

export default {
  title: 'NewComponent',
  component: NewComponent
} as Meta

export const Default: StoryObj = {}
