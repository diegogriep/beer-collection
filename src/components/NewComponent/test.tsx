import { render, screen } from '@testing-library/react'

import NewComponent from '.'

describe('<NewComponent />', () => {
  it('should render the heading', () => {
    const { container } = render(<NewComponent />)

    expect(
      screen.getByRole('heading', { name: /NewComponent/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
