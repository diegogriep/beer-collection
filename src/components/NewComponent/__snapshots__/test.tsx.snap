// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewComponent /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    NewComponent
  </h1>
</main>
`;
